chord_metadata_service/discovery/stats.py

```diff
@@ -2,9 +2,11 @@
 
 from typing import Mapping, Type
 
+from chord_metadata_service.discovery.model_lookups import PUBLIC_MODEL_NAMES_TO_SCOPE_FILTERS
```

Comment:
`from .model_lookups` and group with other local relative imports in alphabetical order